sdk/storage/Azure.Storage.Queues/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.4.0-preview.1 (Unreleased)
-
+## 12.3.1 (2020-06)
+- This release contains bug fixes to improve quality.
```

Comment:
What did we change for Queue?  The .NET framework hanging issue?

Reply:
Yeah, the fix was in the Azure Core so we want to pull that in.